tests/integration/release/conftest.py

```diff
@@ -56,6 +59,12 @@
 
 
 def pytest_addoption(parser: pytest.Parser) -> None:
+    parser.addoption(
+        "--output-dir",
```

Comment:
Remind me - we're mostly punting the problem of how to get all future jobs that are potentially run in parallel on different slurm nodes into this directory?